### PR TITLE
Add a recipe for R package PresenceAbsence

### PR DESCRIPTION
**Pull request by ColineRoyaux**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [ ] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by ColineRoyaux** [comment]
Recipe generated with conda_r_skeleton_helper

File: recipes/r-presenceabsence/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    - r-base
+
+test:
+  commands:
+    - $R -e "library('PresenceAbsence')"           # [not win]
+    - "\"%R%\" -e \"library('PresenceAbsence')\""  # [win]
+
+about:
+  home: https://CRAN.R-project.org/package=PresenceAbsence
+  license: MIT
```
Comment (dbast) [inline recipes/r-presenceabsence/meta.yaml:39]: cran meta-data tells "unlimited"... is there any reference that it is also MIT?

Comment (ColineRoyaux) [reply recipes/r-presenceabsence/meta.yaml:39]: I was certain I saw it somewhere but I can't find it back, found the reference paper for this package though (https://www.jstatsoft.org/article/view/v023i11) it mentions GPL-2.0-or-later for the code, seems right ?

Comment (dbast) [reply recipes/r-presenceabsence/meta.yaml:39]: Wondering if `LicenseRef-unlimited` would make sense?

Comment (ColineRoyaux) [reply recipes/r-presenceabsence/meta.yaml:39]: Maybe, I don't know if custom LicenseRef have any requirements to be used ? Like being defined somewhere in the tarball or online ?

Comment (dbast) [reply recipes/r-presenceabsence/meta.yaml]: Updated it. That is all what is inside https://cran.r-project.org/src/contrib/PresenceAbsence_1.1.9.tar.gz ... linter seems to be happy.

Comment (ColineRoyaux) [reply recipes/r-presenceabsence/meta.yaml]: Ok wonderful ! Thanks :)

**Pull request merged**